spinta/manifests/internal_sql/components.py

```diff
@@ -0,0 +1,79 @@
+from typing import Optional, Dict, List, Final, Literal
+
+from spinta.manifests.components import Manifest
+import sqlalchemy as sa
+
+
+class InternalSQLManifest(Manifest):
+    type = 'internal'
+    path: Optional[str] = None
+
+    @staticmethod
+    def detect_from_path(path: str) -> bool:
+        try:
+            url = sa.engine.make_url(path)
+            url.get_dialect()
+            engine = sa.create_engine(url)
+            with engine.connect() as conn:
+                meta = sa.MetaData(conn)
+                meta.reflect()
```

Comment:
Do we really need to reflect whole database, in order to check single table? If database has many tables, reflect can take minutes to complete.

Please use [Inspector.has_table()](https://docs.sqlalchemy.org/en/20/core/reflection.html#sqlalchemy.engine.reflection.Inspector.has_table).